.github/workflows/pull_requests_redpanda.yaml

```diff
@@ -149,6 +156,13 @@ jobs:
         run: |
           .github/create-sasl-secret.sh "some-users"
 
+      - name: Create license secret
+        if: steps.list-changed.outputs.changed == 'true'
+        env:
+          REDPANDA_LICENSE: ${{ secrets.REDPANDA_SAMPLE_LICENSE }}
+        run: |
+          .github/create-license-secret.sh "redpanda-license" "%REDPANDA_LICENSE%"
```

Comment:
I think this doesn't work as tests shows:

```
2023-09-19T19:20:50.2924650Z >>> kubectl --request-timeout=30s logs redpanda-439416s7xb-configuration-7lkkd --namespace redpanda-439416s7xb --container redpanda-post-install
2023-09-19T19:20:50.4001383Z unable to set license: request PUT https://redpanda-439416s7xb-0.redpanda-439416s7xb.redpanda-439416s7xb.svc.cluster.local.:9644/v1/features/license failed: Bad Request, body: "{\"message\": \"License is malformed: Outer envelope malformed\", \"code\": 400}"
```

https://github.com/redpanda-data/helm-charts/actions/runs/6239380055/job/16938410422?pr=734#step:20:27440

Reply:
Yes I noticed this, was about to start to track down the issue, thanks for looking into it.